internal/common/resource_test.go

```diff
@@ -24,8 +24,9 @@ import (
 var log = logf.Log.WithName("common_operand_package")
 
 const (
-	namespace = "kubevirt"
-	name      = "test-ssp"
+	namespace   = "kubevirt"
+	name        = "test-ssp"
+	changedName = "changed-name"
```

Comment:
The same here. The point of the test is to change the name, not that the changed name is the same in all tests.

Reply:
In a case like this, where the actual content does not matter the way to go seems to be a nameGenerator.
Same as with the IP addresses, generate an arbitrary name for testing.